app/assets/stylesheets/default_theme/table.scss

```diff
@@ -47,7 +47,8 @@
     }
   }
 
-  td {
+  td,
+  > tbody > tr > td {
```

Comment:
Can you explain what this does?
 `td` is always a superset of `> tbody > tr > td`, so this doesn't change the behavior at all IMHO.